src/main/kotlin/racing/domain/Racing.kt

```diff
@@ -1,15 +1,13 @@
 package racing.domain
 
 internal class Racing(
-    private val racingCars: RacingCars
+    private val racingCars: RacingCars,
+    private val randomMovable: RandomMovable
 ) {
-    val cars: List<Car>
-        get() = this.racingCars.cars
-
     val winners: List<Car>
         get() = this.racingCars.winners
 
-    fun nextRound() {
-        racingCars.nextRound()
+    fun race(numberOfRound: Int): List<RacingCars> {
+        return (0..numberOfRound).map { racingCars.nextRound(randomMovable) }
```

Comment:
`race()`에 대한 테스트 코드를 작성해 보면 어떨까요?

```suggestion
        return (1..numberOfRound).map { racingCars.nextRound(randomMovable) }
```